test/TestCommons.sol

```diff
@@ -3,8 +3,11 @@ pragma solidity 0.8.16;
 
 import "forge-std/Test.sol";
 import "../contracts/mocks/tokens/MockTestnetToken.sol";
+import "./utils/factory/IporProtocolFactory.sol";
 
 contract TestCommons is Test {
+    IporProtocolFactory internal _iporProtocolFactory = new IporProtocolFactory(address(this));
```

Comment:
pleas do not use contract address as admin, this cause problems with `deal` cheat code